authors/apps/authentication/serializers.py

```diff
@@ -144,6 +150,11 @@ class Meta:
         # password field, we needed to specify the `min_length` and
         # `max_length` properties too, but that isn't the case for the token
         # field.
+    def get_followers(self, obj):
+        return len(json.loads(json.dumps([u.username for u in obj.followers.all()])))
```

Comment:
Function name should be something like `get_number_of _followers`. Also, for getting the number can we user `count` instead?

Reply:
Let me work on that

Reply:
There must be a better way to do this